cs.html

```diff
@@ -557,6 +706,101 @@ <h2>Dictionary BidirectionalStreamEventInit Members</h2>
         </section>
       </div>
     </section>
+    <section>
+      <h3><dfn>DatagramsReceivedEvent</dfn></h3>
+      <p>The <code><a>datagramsreceived</a></code> event uses the
+      <code><a>DatagramsReceivedEvent</a></code> interface.</p>
+      <div>
+        <pre class="idl">
+        [ Constructor (DOMString type, DatagramsReceivedEventInit eventInitDict), Exposed=Window]
+interface DatagramsReceivedEvent : Event {
+    readonly        attribute sequence<Uint8Array?> datagrams;
+};</pre>
+        <section>
+          <h2>Constructors</h2>
+          <dl data-link-for="DatagramsReceivedEvent" data-dfn-for="DatagramsReceivedEvent"
```

Comment:
I think you need to remove all the DatagramsReceivedEvent/DatagramsReceivedEventInit stuff.

Reply:
Done